src/pages/User-Infor/index.vue

```diff
@@ -0,0 +1,316 @@
+<script setup>
+import { onBeforeMount, reactive } from 'vue';
+import { validateFullName, validateDOB, validataPhoneNumber, validatePassword } from '@/utils/validator';
+import { getUserById, updateUserAddress, updateUserInformation, changeUserPassword, deleteUserById } from '@/api/user.api';
+import { useAuthStore } from '@/stores/auth';
+import { useGeolocation } from '@vueuse/core';
+import router from '@/router';
+import { collapseLight } from 'naive-ui/es/collapse/styles';
+
+const message = useMessage();
+const auth = useAuthStore();
+const loadingBar = useLoadingBar();
+const disabledRef = ref(true);
+const dialog = useDialog()
+const userForm = ref({});
+const user = ref({});
+const account = reactive({
+    username: "",
+    oldPassword: "",
+    password: "",
+    repeatPassword: ""
+})
+const customerId = ref(-1);
+const loadCustomer = async () => {
+    if (!(typeof auth.auth.customerId === 'undefined') || !(typeof auth.auth.customerName === 'undefined')) {
+        customerId.value = auth.auth.customerId;
+        const data = await getUserById(customerId.value);
+        userForm.value = data.data;
+        let timestampDob = new Date(userForm.value.dob).getTime();
+        userForm.value.dob = timestampDob;
+        user.value = {
+            fullName: data.data.fullName,
+            phoneNumber: data.data.phoneNumber,
+            address: data.data.address.addressName
+        };
```

Comment:
Dùng ref rồi thì gán bằng cả cái data điiii
```suggestion
        user.value = data.data
```

Reply:
khum được, làm thế gây ra tường hợp thay đổi bên phải, bên trái cũng thay đổi theo, như sử dụng một biến như lần đầu luôn 
![image](https://github.com/phuonghight/HIT-FOOD-Client-Vue/assets/118819551/7d9f603f-23c8-4b25-b9f4-29eae6a5809d)
![image](https://github.com/phuonghight/HIT-FOOD-Client-Vue/assets/118819551/73ea70e9-a798-4a1e-bb38-8a17c192b60b)
--> Chưa nhấn thay đổi đâu

Reply:
vậy thì dùng spread operator
```suggestion
        user.value = { ...data.data }
```